### PR TITLE
Add 2024 September Apple Event

### PR DESCRIPTION
**Pull request by Dreel0akl**
added the new sep 9th 2024 apple event

### CODE REVIEW

**Comment by Gr8z** [comment]
Please change the title to - "It's Glowtime"

**Comment by Dreel0akl** [comment]
fixed.

File: js/addNewEvent.js [hunk 1]
```diff
@@ -6,16 +6,16 @@
  *
  */
 const // The time zone when the event will be held. Format: time zone identifier (e.g., "PDT" or "PST")
-	timeZone = 'PDT',
+	timeZone = 'PT',
```
Comment (andrewheavin) [inline js/addNewEvent.js:9]: Does timeZone need to remain PDT as mentioned in the comments above?

File: js/addNewEvent.js [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
 	// The month as a number, not the index
 	// Format: MM (09) or M (9), both are valid
-	month = 06,
+	month = 09,
 	// Format: DD (09) or D (9), both are valid
-	day = 10,
+	day = 09,
 	// Format: 0 - 24 / 5 == 5 AM / 17 == 5 PM
-	hour = 10,
+	hour = 18,
```
Comment (andrewheavin) [inline js/addNewEvent.js:18]: This should remain 10.

**Comment by Dreel0akl** [comment]
should hopefully be fixed enough to be pushed i think

**Comment by andrewheavin** [comment]
Looks good to me!

File: js/addNewEvent.js [hunk 3]
```diff
@@ -23,4 +23,4 @@ const // The time zone when the event will be held. Format: time zone identifier
  * UPDATE UPCOMING EVENT NAME
  * --------------------------
  */
-const eventName = 'Apple WWDC24 - World Wide Developers Conference';
+const eventName = 'Its Glowtime.';
```
Comment (hartator) [inline js/addNewEvent.js:26]:
```suggestion
const eventName = "It's Glowtime.";
```

**Pull request merged**